.prettierignore

```diff
@@ -3,4 +3,4 @@ lib/*/
 conan-pkgs/*/
 cmake/sanitizers-cmake/
 
-.github/
+.clang-tidy
```

Comment:
If we ignore `.clang-tidy` (which I don't think we should do long-term, but I'm ok with that being made in a followup PR), we should also ignore `.clang-format`

Reply:
I included the `.clang-format` files in this PR.